src/plugins/irc-events/error.js

```diff
@@ -59,8 +59,7 @@ module.exports = function(irc, network) {
 		lobby.pushMessage(client, msg, true);
 
 		if (irc.connection.registered === false) {
-			const random = "i" + Math.random().toString(36).substr(2, 10); // 'i' so it never begins with a number
-			irc.changeNick(random);
+			irc.changeNick("thelounge" + Math.floor(Math.random() * 100));
```

Comment:
This reduces confusion by going to something that looks like our default fallback.

I debated bringing this from the default config file itself, but that would have meant moving code around to put [this](https://github.com/thelounge/thelounge/blob/5738642d44c1f4c2dd44b1f67dca35bc8c84c96a/src/server.js#L572) into a shared location, but I figured it was too much of a hassle at 3am for such a simple change.
Happy to continue the improvement, but that'd be in a further PR. Where would it be a reasonable shared location?